src/utils/utils.ts

```diff
@@ -22,7 +22,7 @@ export function useNetworkId(): [NetworkId, (network: NetworkId) => void] {
   const [network, setNetwork] = useLocalStorage<NetworkId>({
     key: networkStorageKey,
     defaultValue: 'mainnet',
-    getInitialValueInEffect: true,
+    getInitialValueInEffect: false,
```

Comment:
The following is the reason for rendering twice:

1. If `getInitialValueInEffect` is true, it will first return the `defaultValue` and then read the value from storage. If the `defaultValue` differs from the value in storage, it will lead to rendering twice.
2. If `getIntialValueInEffect` is false, it will directly read from storage, and it returns the `defaultValue` if the storage does not exist, so it will only render once.